### PR TITLE
Make criteria curves accesible as part of the mapca object

### PR DESCRIPTION
**Pull request by eurunuela**
Closes none.

This PR makes the `aic`, `kic` and `mdl` criteria curves accessible as `mapca.aic`, `mapca.kic` and `mapca.mdl`.

The main purpose of this PR is to give users the chance to look at the optimization curves.

### CODE REVIEW

File: mapca/mapca.py [hunk 1]
```diff
@@ -94,6 +94,12 @@ class MovingAveragePCA:
 
         Equal to the average of (min(n_features, n_samples) - n_components) smallest
         eigenvalues of the covariance matrix of X.
+    aic: array, shape (n_components)
```
Comment (tsalo) [inline mapca/mapca.py:97]:
```suggestion
    aic : :obj:`numpy.ndarray`, shape (n_components)
```

File: mapca/mapca.py [hunk 2]
```diff
@@ -94,6 +94,12 @@ class MovingAveragePCA:
...
+        The Akaike Information Criterion optimization curve.
+    kic: array, shape (n_components)
```
Comment (tsalo) [inline mapca/mapca.py:99]:
```suggestion
    kic : :obj:`numpy.ndarray`, shape (n_components)
```

File: mapca/mapca.py [hunk 3]
```diff
@@ -94,6 +94,12 @@ class MovingAveragePCA:
...
+        The Kullback-Leibler Information Criterion optimization curve.
+    mdl: array, shape (n_components)
```
Comment (tsalo) [inline mapca/mapca.py:101]:
```suggestion
    mdl : :obj:`numpy.ndarray`, shape (n_components)
```

**Review by tsalo — CHANGES_REQUESTED** [review]
I noticed some minor docstring formatting things. Also, I think that sklearn's convention for attributes defined by the `fit` method is to include a trailing underscore in the name (e.g., `self.aic_`. I think it might be nice to adopt that convention, since many of our users will also be sklearn or nilearn users, but if you don't want to that's fine too.

**Comment by eurunuela** [comment]
> I noticed some minor docstring formatting things. Also, I think that sklearn's convention for attributes defined by the `fit` method is to include a trailing underscore in the name (e.g., `self.aic_`. I think it might be nice to adopt that convention, since many of our users will also be sklearn or nilearn users, but if you don't want to that's fine too.

I didn't know about that. Thank you @tsalo !

**Review by tsalo — APPROVED** [review]
LGTM!

**Pull request merged**